### PR TITLE
Various cleanups and some improvements in theme

### PR DESCRIPTION
**Pull request by thaJeztah**
- Reformat HTML of the landing-page
- Include code-highlight styles in main stylesheet so that it isn't loaded separately
- Remove/disable unused highlighjs styles that were still present
- Remove unused menu.js script
- Remove unused modernizr script
- Remove broken "NOT_EDITED_HERE" symlink

### CODE REVIEW

**Comment by thaJeztah** [comment]
Splitting this off from the branch I was working on for https://github.com/docker/docker.github.io/pull/11524

@StefanScherer @usha-mandya PTAL

File: _includes/body-landing.html [hunk 1]
```diff
@@ -129,52 +123,18 @@ <h5 class="title">How do I?</h5>
...
-            </div>
-            <div class="col-xs-12 col-md-6">
-              <a href="/get-started/swarm-deploy/">
-                Scale apps using Swarm
-              </a>
-            </div>
+            <div class="col-xs-12 col-md-6"><a href="/engine/reference/builder/">Write a Dockerfile</a></div>
+            <div class="col-xs-12 col-md-6"><a href="/network/">Manage container networking</a></div>
+            <div class="col-xs-12 col-md-6"><a href="/compose/compose-file/">Write a Docker Compose file</a></div>
+            <div class="col-xs-12 col-md-6"><a href="/storage/">Work with volumes and bind mounts</a></div>
```
Comment (thaJeztah) [inline _includes/body-landing.html:129]: I thought having a link per-line was easier to use; even though some lines are now a bit long, it gives an easier overview how many links are in a section, and to add/remove a link is now just removing a line (or copying one), without having to find where a link block "starts" and "ends", so less chances on making mistakes

**Review by StefanScherer — APPROVED** [review]
The commits look good to me.

**Review by usha-mandya — APPROVED** [review]
Thanks for cleaning these up!

File: _includes/body-landing.html (hunk 1 above)
Comment (usha-mandya) [reply _includes/body-landing.html:129]: Thanks. Yes, this makes sense.

**Pull request merged**